### ISSUE TITLE
Windows - Problem with breakpoints

### ISSUE THREAD

**Issue by zerabba** [NicolasPetton/Indium#256]
Hi,

Thanks for your work! 

I tried to debug my project with chrome and I had an issue to add breakpoints. I found that you added a capitalize for the drive letter on the emacs side, but in Indium, the drive letter is in small. 

```lisp
(defun indium-client--convert-path (path)
  "Convert PATH to a system path that the server component understands."
  (when (eq system-type 'windows-nt)
    (setq path (replace-regexp-in-string "/" "\\" path nil t))
    (setq path (replace-regexp-in-string "^\\([a-z]\\):" #'capitalize path)))
  path)
```

I've commented the line in emacs and it's now working! Why is it capitalized? Maybe it's possible to make a case insensitive match in indium?

**Comment by NicolasPetton** [comment]
Thanks for figuring that out.

Since I'm not using Windows very often, it might just be a leftover. Could you open a pull request with your fix?

Thanks!

**Comment by NicolasPetton** [comment]
Fixed in 32487a4.
